src/components/Global/Modal/index.tsx

```diff
@@ -18,6 +18,7 @@ type ModalProps = {
     classNameWrapperDiv?: string
     showPrev?: boolean
     onPrev?: () => void
+    hideCloseButton?: boolean
```

Comment:
Q: Why do we need a hide close button option?

Reply:
was planning to move the modal to look like sheet for mobile screens, figured that's a little lengthy task and not high priority, for now reverted it to old structure, tracking it to and will tackle in another spirint